### ISSUE TITLE
Inspect opens an empty window

### ISSUE THREAD

**Issue by samhed** [googlearchive/ADBPlugin#14]
**EDIT by admin December 2014**: The following [information](https://github.com/GoogleChrome/ADBPlugin/issues/14#issuecomment-27277230) from @gockxml is very helpful for developers in China:

> #### 致中国同胞：如果你按照要求操作最后却得到一个空窗口，翻墙后再试试。不用谢。
> 1. Remote debugging with Chrome downloads files from https://chrome-devtools-frontend.appspot.com. 
> 2. **Test your connection:** Try to open https://chrome-devtools-frontend.appspot.com 
> 3. If it doesn't work, thats why you're getting an empty window.
>    1. In this case maybe you need a VPN. 
> 
> This is really a problem for people in China because the appspot.com domain is blocked by [GFW](http://en.wikipedia.org/wiki/Great_Firewall_of_China). So sad.

_(original issue follows…)_

<hr>

<hr>

<hr>

When clicking "inspect" next to the url of the page open on the Nexus 7 an empty window shows up and nothing happens.

![screenshot from 2013-07-03 12 22 35](https://f.cloud.github.com/assets/4117162/742464/d6ebb220-e3ca-11e2-839b-3461dc0d8b18.png)

ADBPlugin (0.9.6) in Chrome (Version 28.0.1500.89 beta) running on Fedora 18.

**Comment by MichaelRFairhurst** [comment]
I am having this issue as well! Please let me know if you find a solution.

On Fedora 17, chrome stable 28, nexus 4 (jellybean), adbplugin 0.9.6

Here is the data being passed to chrome.send('inspect', data); bound to the inspect link's onclick

adbDebugUrl: "/devtools/page/1"
adbFrontendUrl: "https://chrome-devtools-frontend.appspot.com/static/18.0.1025.166/devtools.html?host=&page=1"
adbModel: "Nexus 4
↵"
adbPackage: "Chrome"
adbSerial: "01b6f5d40a61745f"
adbSocket: "chrome_devtools_remote"
attached: false
faviconUrl: "http://192.168.3.50/favicon.ico"
name: ""
pid: 0
processId: 0
routeId: 0
type: "adb_page"
url: "http://192.168.3.50/#fasttrack/1"

**Comment by sam3k** [comment]
same here. Worked for a few minutes and then i got the blank popup.

**Comment by ghost** [comment]
Same here. Not one successful debug session yet after a day filled with joy.

Also noticed that it loses connection with the device roughly every minute.

Version 29.0.1547.15 dev on MacOS 10.8.4, targeting Nexus 4 running 4.2.something (sorry, device went out window)

**Comment by doochik** [comment]
+1
I see some errors in insepct for the remote debug console

```
Uncaught TypeError: Object #<Object> has no method 'setToolbarColors'
Uncaught TypeError: Object #<Object> has no method 'setDockSide'
```

OSX 10.8.4, chrome 28.0.1500.71 beta, try to debug nexus 7 4.2.2

**Comment by hrussell** [comment]
I'm experiencing the same problem.  Has anyone figured this one yet?

**Comment by ginchly** [comment]
Same for me unfortunately - looking forward to trying this out though!
OSX 10.7.5, Version 30.0.1568.0 canary, adbplugin 0.9.6, nexus 4 running 4.2.2

**Comment by lukeedmonds** [comment]
Same error too, hopefully someone will figure it out soon enough.

**Comment by bullgare** [comment]
same error. Mac OS X 10.7.5.
I can provide extra data on demand.

**Comment by dhharker** [comment]
I was experiencing this, linux mint 13, 29.0.1547.22 beta + adb 0.9.6.

FIXED: Installed latest version of Chrome (HTC One S) from Play and it has started working.

**Comment by stuartmemo** [comment]
Same here on 30.0.1581.2 canary osx 10.8.4 trying to debug Chrome 18.0.1025469 on Nexus 7.

Seen a few reviews on the Chrome Web Store seeing this same bug.

**Comment by ericente** [comment]
Had this problem on OSX 10.8.4 and Desktop Chrome Beta 29. Resolved by updating to the latest version of mobile Chrome on the Android device (Nexus 10)

**Comment by paulirish** [comment]
This error will manifest if your Chrome on Android is v18. You will need to
update it from Play to get it working.

**Comment by ginchly** [comment]
Ah of course - thanks Paul, this fixed it for me.

**Comment by donaldr** [comment]
The instructions (https://developers.google.com/chrome-developer-tools/docs/remote-debugging) categorically state that Chrome on your phone has to be at least version 28.  Check the version of your Chrome. If it's anything less then update via the play store.

**Comment by paulirish** [comment]
@donaldr we updated them recently to be a bit more clear as a result of this issue thread. :)  
The docs got other updates that should make all this much easier, but we're still improving things and in a month or so I hope to share some of the improvements to remote debugging we've been working on.

**Comment by kaznacheev** [comment]
Good news everyone! This problem is fixed in Chrome 31.0.1616.0.

**Comment by gockxml** [comment]
FYI, the last step needs to visit https://chrome-devtools-frontend.appspot.com. If you cannot open the link, that's why you got an empty window. In this case maybe you need a VPN. This is really a problem for people in China because the appspot.com domain is blocked by [GFW](http://en.wikipedia.org/wiki/Great_Firewall_of_China). So sad.

致中国同胞：如果你按照要求操作最后却得到一个空窗口，翻墙后再试试。不用谢。

**Comment by lazurey** [comment]
Thank you,  gockxml.

Remember to add proxy for _HTTPS_

**Comment by yrom** [comment]
@gockxml Thanks!

**Comment by taxusyew** [comment]
@gockxml that's it!

**Comment by zhjuncai** [comment]
GFW = Great F**\* Wall

Thanks @gockxml It helps but the ajax couldn't access my server behind proxy :-(

**Comment by sansmash** [comment]
Same issue here. But I'm having version 32 in both mobile and computer as well.

**Comment by bullgare** [comment]
@sansmash they say you don't need it in chrome 32 anymore - http://youtu.be/gZH1d2Co1X0?t=1m48s

**Comment by Archieeeeee** [comment]
@gockxml thanks

**Comment by Sawyer51** [comment]
@paulirish  I am developer in China, with VPN,  I  still got a blank window after inspect?
I used the updated versions of Chrome canary and Chrome Beta. Please, help. Thanks.  
I seems like all the contents that supposed in the windows  stay in the title.

**Comment by paulirish** [comment]
quoting @gockxml …

> FYI, the last step needs to visit https://chrome-devtools-frontend.appspot.com. If you cannot open the link, that's why you got an empty window. In this case maybe you need a VPN. This is really a problem for people in China because the appspot.com domain is blocked by [GFW](http://en.wikipedia.org/wiki/Great_Firewall_of_China). So sad.
> 
> 致中国同胞：如果你按照要求操作最后却得到一个空窗口，翻墙后再试试。不用谢。

Unfortunately this is still the case. We're looking at solutions.

**Comment by zhjuncai** [comment]
Great @paulirish 

I hope one day in China we don't need VPN at all to use google services, that will be awesome!!

**Comment by igo903** [comment]
@gockxml  Thank you for finding the root cause. Goagent can help on this.

**Comment by paulirish** [comment]
@igo903 can you write up a guide on how to use goagent to resolve this problem?

**Comment by mrdumont** [comment]
The remote debugger had been working perfectly for me until an update to chrome beta v34.01.1847.36 yesterday. Now I'm experiencing the blank screen. I am able to open https://chrome-devtools-frontend.appspot.com/ but tried a VPN anyway just in case. No change, still not working.

**Comment by travisholliday** [comment]
Sadly, I am having this issue now as well. In the same boat as @mrdumont .

**Comment by yangjunjun** [comment]
@gockxml  Thanks.
And Fuck GFW.

**Comment by pauloneves** [comment]
Same has @mrdumont 
Failed with Chrome Beta, but is working with the default chrome. My host machine is Ubuntu 13.10

**Comment by MoritzGlantz** [comment]
Same here. Chrome 33.0.1750.154 m on Win7 and Chrome 34.0.1847.114 on my Nexus 5. It worked some versions ago.

**Comment by AzazelN28** [comment]
Yep, same here. Chromium 33.0.1750.152 Ubuntu 13.10 and Chrome 34.0.1847.11 on Nexus 4. This morning it was working fine, so I tried to fix this by my self (yup, I'm very optimistic) and in the Developer Tools of the Developer Tools it shows me these errors:

Uncaught TypeError: Object #<Object> has no method 'setDockSide'
Uncaught TypeError: Object #<Object> has no method 'setAttachedWindow'
Uncaught TypeError: Object #<InspectorFrontendHost> has no method 'zoomFactor' 

And I think that for Object #<Object> it means InspectorFrontendAPI. I hope this could help.

**Comment by kevinorin** [comment]
Been troubleshoothing this for way too long. Was working fine, yesterday stopped working so I updated all my Chrome versions on mobile and Macbook and still nothing. SG S3 Chrome version 34.0.1847.114 - It flickers a bit as if it's going to work then nothing.

**Comment by paulirish** [comment]
If you have Chrome 34 on your device you'll need to debug it with a desktop
browser that is at least 34.

**Comment by AzazelN28** [comment]
@paulirish Thanks.

**Comment by kevinorin** [comment]
But are both desktop + mobile on 34+? - I didn't check desktop version but I did update both to latest just to be sure and still doesn't work.

**Comment by timcooper** [comment]
The current Windows stable is 33 and Android 34. Windows dev channel is 35 and beta 34, does this mean we need to switch to Windows beta channel? Android Beta is a lower version than Android Stable. I'm struggling to find an easy reference for what version is current in which channel, there doesn't seem to be a clear list anywhere - having to hunt through http://googlechromereleases.blogspot.co.uk/ is a bit of a pain. Found http://omahaproxy.appspot.com/ too but it seems to be out of date, has android stable listed as v33 still.

**update:** Switching to Chrome Beta channel (currently at v34 which matches android stable) fixes the issue, this is going to keep cropping up though with differing stable releases though.

**Comment by MoritzGlantz** [comment]
Thanks @paulirish. Chrome 36 (canary) and Chrome 34 on the mobile device works.

**Comment by elboletaire** [comment]
I cannot get it working using Windows 7 and the latest versions of google chrome for Windows (33.0.1750.154) and for Android (34.0.1847.114)...

**Comment by MoritzGlantz** [comment]
Yup. That was my problem as well. Use at least Chrome 34 on your desktop.

**Comment by elboletaire** [comment]
I'm not gonna update my chrome to a beta version... 

In fact, I think that this should be specified [in the official documentation](https://developers.google.com/chrome-developer-tools/docs/remote-debugging)

**Comment by kurehajime** [comment]
@paulirish Thanks.

**Comment by bytesphere** [comment]
Thanks @timcooper, switched to Chrome Beta (34.0.1847.116) on OSX. Now remote inspect is working again with Nexus 4 and Chrome 34 stable.

**Comment by stroep** [comment]
@elboletaire You can have both the stable and the canary version installed on your machine. Works perfectly

**Comment by mgenev** [comment]
seems to happen when mobile chrome and desktop chrome are out of sync.  Apparently version 34 auto-updated on my phone while desktop still had 33 and my dev tools stopped working.

Please fix guys

**Comment by paulirish** [comment]
Chrome desktop 34 just shipped yesterday. Update and you should be all set.

I apologize for the incompatibility between 33 and 34. We're going to try to avoid that in the future.

**Comment by marlencrabapple** [comment]
This is a bit of a problem for people using stable Linux distros. Chromium 34 hasn't been added to many repos yet.

**Comment by paulirish** [comment]
@timcooper Saw your comment then I think you deleted it? Is your Chrome stable on 35?

**Comment by timcooper** [comment]
Yes my bad, I did delete it. I'm having a one of those 4-30-5pm Friday everything going to hell type moments and my comment came across a bit angrier than I was intending.

It was also incorrect, the issue turned out to be the other problem where it just doesn't see the phone, I ran into it once before but I noticed the version mismatch before I realised and assumed it was that.

As before it was fixed by running `adb usb` - happy to report that I can debug android v34 from my desktop v35 now.

**Comment by hkongm** [comment]
@gockxml 牛逼！！

**Comment by scoleman2272** [comment]
I'm using Chrome V35.0.1916.141 on my mobile (kitkat), and V35.0.1916.153 on my mac and still get the blank screen - very frustrating!

**Comment by AdrianTP** [comment]
So I'm not clear on one thing here...if I cannot access the internet from my laptop, I cannot run the remote debugger's inspector against my phone's browser? I'm connected to a WiFi network which does not have internet access, and I am running a webserver on my laptop. I can access the website on my phone (which is connected to the same network as my laptop) and my laptop's Chrome lists my device and open tabs in the Inspect Devices page, yet clicking Inspect opens a blank window. I have the newest release of Chrome on both my phone and my laptop. I tried with both Windows and OS X.

**Comment by raytaylorlin** [comment]
@gockxml Thank you very much! Exactly the GFW!

**Comment by paulirish** [comment]
@AdrianTP the inspector gets cached in appcache after first request, so typically folks are fine. If you can't access internet on your laptop then we don't have a good solution for that right now.

**Comment by laosiaudi** [comment]
@gockxml  seems not for my situation......It still opens an empty window when opening an Android-Crosswalk App, but succeeds when opening an Android-Cordova App

**Comment by jimmyxu0311** [comment]
I'm using Chrome V38.0.2125.102 on my mobile (kitkat), and v38.0.2125.101 m on my win7 and still get the blank screen - very frustrating!

**Comment by hkongm** [comment]
@jimmyxu0311 哥们，上面提到了，需要翻墙哦

**Comment by gooooloo** [comment]
@gockxml 你救了我

**Comment by Perlesvaus** [comment]
The same for me (W7 x64) address instead of title: chrome-devtools://devtools/bundled/devtools.html?&remoteFrontend=true&remoteFrontendUrl=chrome-devtools

**Comment by p2227** [comment]
@gockxml 除了翻墙真的没有其他方法了吗？其实访问那个网站是要什么信息，能不能本地搭建相关的服务器。

**Comment by juliogc** [comment]
I'm using Chrome 39.0.2171.71 and Chrome Canary 41.0.2243.0 in Mac OS 10.10.2 and still getting the blank screen in both.

**Comment by Perlesvaus** [comment]
juliogc, try to "revoke" the USB debugger inside Settings and re-pair it. Let me know if it works!

**Comment by juliogc** [comment]
Perlesvaus, in Canary back to normal, but in Chrome still blank.

**Comment by michalliu** [comment]
@gockxml Thank you
Debugging chrome browser is okay now, But I got a blank page when debugging webview on the same phone, xiaomi 3, why?

**Comment by keepeye** [comment]
i have the same problem.Even though i used shadowsocksx on osx,it is still so slow,it costs about 10 minutes when inspect one page

**Comment by vaenow** [comment]
@gockxml Thank you for saving my life!

**Comment by rlynd3** [comment]
@paulirish Anyway to host the front-end "locally"? as you used to be able to with the "--remote-debugging-frontend" flag for desktop. Happy to recompile chromium if need be...

**Comment by martin-s** [comment]
Any news on debugging webviews yet? I still recognize requests to "https://chrome-devtools-frontend.appspot.com/" which obviously are not going to work in an offline environment.

**Comment by paulirish** [comment]
@martin-s 

> Any news on debugging webviews yet? 

Been working for a while! https://developer.chrome.com/devtools/docs/remote-debugging#debugging-webviews

>  I still recognize requests to "https://chrome-devtools-frontend.appspot.com/" which obviously are not going to work in an offline environment.

There's an appcache so after you access the files for the first time, the frontend will come from there. Works fine offline.

@rlynd3

> Anyway to host the front-end "locally"?

There's a good new solution for that. http://sandipchitale.blogspot.com/2014/07/devtoolsapp-now-works-with-chrome-360.html offers a Chrome app where you can chose which frontend to attach to your debugging target. You can supply one locally or use another host. 
cc @sandipchitale

**Comment by rosone** [comment]
I still recognize requests to "https://chrome-devtools-frontend.appspot.com/" which obviously are not going to work in an offline environment.

this is the answer.

after I connect America vpn, it works.

**Comment by bmwertman** [comment]
Just installed Chrome Canary Version 42.0.2283.5 on Mac OSX Yosemite 10.10.1 I'm getting the blank page when I click inspect device.

**Comment by Elijen** [comment]
I have the same problem (blank page) on 41.0.2272.35 and 42.0.2288.6 both Yosemite 10.10. Android running the WebView is version 4.4.4

I doubt that my ISP is blocking appspot.com or any other resources (I reside in Spain).

**Comment by landys** [comment]
Thanks, it solved my problem. :)

**Comment by mrzzcn** [comment]
使用这个工具: https://github.com/huhamhire/huhamhire-hosts 更新最新的Google IP 
下面这个是最近更新的.
64.233.162.83       chrome-devtools-frontend.appspot.com
每次更新完成之后, 查找Google.com对应的IP,把chrome-devtools-frontend.appspot.com这个域对应到那个IP.

**Comment by qiushilee** [comment]
@mrzzcn Thank you very much!

**Comment by g8up** [comment]
能不能把 chrome-devtools-frontend.appspot.com 这个请求返回的 JSON 在本地搭建 http server + hosts 来模拟一下？

**Comment by jhilmar-sergio** [comment]
it is working with the default chrome (after install canary, I don't know wy)

**Comment by bguiz** [comment]
I'm still experiencing the same issue, on the current versions. 
- Not working on Chromium 42 on Mac OSX
- Working on Chrome 43 on Mac OSX
- Working on Chromium 41 on Ubuntu Linux

Both using the same Android phone, using the same app (phonegap/ cordova)

appspot.com is **not** blocked for me.

**Comment by luke-j** [comment]
Can confirm this issue is still present for me.
- Not working on Chrome 43, Mac OSX.
- Working on Chome 45, Mac OS X.

Android device is up-to-date and appspot is not blocked.

**Comment by jcuenod** [comment]
Likewise not working:
- Chromium 43.0.2357.81 on FC22

**Comment by liuyanghejerry** [comment]
Thanks for the fix. It works for me.

43.0.2357.132 dev-m (64-bit)

**Comment by escapedcat** [comment]
omg @gockxml thx a bunch! that was fun... m(

Still had it with:

```
Version 44.0.2403.61 beta (64-bit)
Linux Mint 17
```

But after opening it one time using a VPN it seems to be fine after even without a VPN.

**Comment by 1e0ng** [comment]
:+1: @gockxml

**Comment by Tyguy7** [comment]
I have this issue as well.  I'm shocked at how shoddy the chrome debugging tools are.  I can't get them to work on Yosemite, or Windows 7, with either my android tablet OR my android phone....  real failures here Google.

**Comment by lifei** [comment]
@gockxml 作为身处墙内的大陆同胞，非常感谢。

**Comment by pallyoung** [comment]
@gockxml 墙内人民发来问候

**Comment by Hanruis** [comment]
It help me a lot ...and I find that if your Inspector has mess layout, or some other things wrong , you yan open  `chrome://appcache-internals/` and clear the manifest cache , then reload the resource

**Comment by ace-han** [comment]
@gockxml You save my days in China!!! Very good job, thxxxxxxx!

**Comment by khejing** [comment]
@Hanruis I use VPN and still get nothing, after I clear all the cache, then it's back again! Really thank you!

**Comment by hakib** [comment]
I got the same with Chrome Version 48.0.2564.97 (64-bit) on Ubuntu 15.10 trying to inspect a cordova 5.4 app with crosswalk. After removing crosswalk it worked fine. P.S, same version on windows 7 (including crosswalk) work fine as well.

**Comment by MysteryAngle** [comment]
Fuck GFW Again.

**Comment by SystemDisc** [comment]
I have this issue. If I open inspector, a blank inspector opens with the following in the titlebar:
```
chrome-devtools://devtools/bundled/inspector.html?isSharedWorker=true&remoteBase=https://chrome-devtools-frontend.appspot.com/serve_file/@f87f641eOabefed98578934of1d5ca79651bf82/&dockSide=undocked
```

macOS Sierra 10.12.3, Chrome 57.0.2987.98 (64-bit)

**Comment by EngleVV** [comment]
It works, thanks.

**Comment by bbuchalter** [comment]
Still an issue for me. macOS 10.12.5, Version 59.0.3071.115 (Official Build) (64-bit). This thread is quite long. Is anyone able to summarize the latest on the root causes and workarounds?

**Comment by escapedcat** [comment]
@bbuchalter the most common solution for this seems to be a VPN. Because the most common reason to this seems to be related to being in China.
I don't know your individual situation, but if you are not in China it might be related to the specific network setup you are working in.

**Comment by zdarovka** [comment]
@bbuchalter I believe it has nothing to do with network, I was not on VPN and I am not located in China, and I haven't done anything with my network and it just stopped doing that, thank god.  believe it is a random problem

**Comment by sigsegv8** [comment]
I also get the same issue with an Android 7 device with chrome 59.0.3071.125. My mac has OSX 10.12.5 and has chrome browser version  59.0.3071.115 (Official Build) (64-bit).

All I see is popup that is a empty/blank.

**Comment by SystemDisc** [comment]
@escapedcat This seems to have nothing to do with being in China (or any specific network for that matter).... I'm in the US - this is just a bizarre bug with Chrome that sometimes (and it's hard to reliably reproduce) results in a second, empty inspector window opening when you open inspector.

To quote my earlier message which has more detail:
> I have this issue. If I open inspector, a blank inspector opens with the following in the titlebar:
> ```
> chrome-devtools://devtools/bundled/inspector.html?isSharedWorker=true&remoteBase=https://chrome-devtools-frontend.appspot.com/serve_file/@f87f641eOabefed98578934of1d5ca79651bf82/&dockSide=undocked
> ```
> 
> macOS Sierra 10.12.3, Chrome 57.0.2987.98 (64-bit)

The last time this was happening to me, I tried disabling all extensions and it did not resolve the issue.

**Comment by bmlyon** [comment]
I too am having the issue SystemDisc described, except on a Windows 10 x64 PC from within the US.

**Comment by astanb** [comment]
This is happening to me when trying to debug a Cordova web view running on Android 5.0.1 (Chrome 42.0.2311.138) - Other Android versions 6+ are fine.

**Comment by rickyk586** [comment]
Updating chrome worked for me. Others have stated that there's an issue when the versions differ between mobile and desktop

**Comment by tperrelli** [comment]
Hello guys! I've got this issue today.

I did the following steps to solve:

1. Just open chrome://chrome-urls/
2. Search for "for debug" section.
3. Access chrome://restart/ url

chrome://restart/ might be different from restarting the pc or simple closing and reopening chrome like a normal user (i don't know why).
I believe that this command really force to restart all processes or something like that!?

Anyways, i use Mac OSX and now i'm abble to debug again. See ya!

**Comment by reyx** [comment]
For me the solution was update chrome to the latest version 65.0.3325.162 (Windows)

**Comment by datencia** [comment]
same over here... yesterday it was working ok, magically not today. Fixed upgrading chrome to 65.0.3325.162

**Comment by ctrl-alt-supr** [comment]
Interestingly, I could inspect apps I had previously inspected but not newly created ones.
As reyx and datencia suggested upgrading from 64 to 65 of chrome has fixed the issue for me and I can inspect as usually right now. TY guys!

**Comment by DrMabuse23** [comment]
thx @tperrelli save my day

**Comment by NicolasBehra** [comment]
On ubuntu I just installed google-chrome in BETA:

> sudo apt install google-chrome-beta

and the dev-tools work again like a charm
Hope it helps :)

**Comment by iamjoyce** [comment]
@reyx Thank you. Goodness me, there was no prompt on Google Chrome to say that the version was outdated. [I had to download the exe instead.](https://www.google.com/chrome/)

**Comment by rasoulmiri** [comment]
Check this ways:
1. update chrome in mobile and in windows or mac
2. use VPN for connect to server google (China, Iran,...)

**Comment by Black-Kitten** [comment]
我在检查器里面能够看见dom节点 但是里面都是空的 除了script标签
有大佬能够给我什么帮助吗。。
![image](https://user-images.githubusercontent.com/34472825/52545026-5c457400-2def-11e9-84f9-d36b388f3f25.png)

**Comment by LycheeHe** [comment]
I know this issue posted long time ago, but I have to say 
  this answer(Remote debugging with Chrome downloads files from https://chrome-devtools-frontend.appspot.com.
Test your connection: Try to open https://chrome-devtools-frontend.appspot.com
If it doesn't work, thats why you're getting an empty window.
In this case maybe you need a VPN.) 
let my confusion go, I try many times to reload my android app and click 'inpect' button from DevTools in chrome. 

Thanks a lot

**Comment by dccat1** [comment]
Any solution now? I had tried all solutions i can find on google but got the same blank

**Comment by robberfree** [comment]
@dccat1  No easy way.I tried different ways last week.The easiest way maybe setup a network that can access google service.

**Comment by metinsevindik** [comment]
> FYI, the last step needs to visit https://chrome-devtools-frontend.appspot.com. If you cannot open the link, that's why you got an empty window. In this case maybe you need a VPN. This is really a problem for people in China because the appspot.com domain is blocked by [GFW](http://en.wikipedia.org/wiki/Great_Firewall_of_China). So sad.
> 
> 致中国同胞：如果你按照要求操作最后却得到一个空窗口，翻墙后再试试。不用谢。

I was able to open this url. But still not working.
Is there any solution ?
